Cartfile

```diff
@@ -1,6 +1,6 @@
 github "swisspol/GCDWebServer" ~> 3.3.3
-github "SnapKit/SnapKit" ~> 3.0.1
+github "SnapKit/SnapKit" "swift-4"
 github "adjust/ios_sdk" ~> 4.10.2
-github "cezheng/Fuzi" ~> 1.0.1
-github "mozilla-mobile/AutocompleteTextField" "master"
+github "cezheng/Fuzi" "master"
+github "boek/AutocompleteTextField" "carthage"
```

Comment:
is there a way to get this back on mozilla-mobile?

Reply:
Oh yes, I couldn't commit to the `mozilla-mobile` branch. Will open a PR

Reply:
mozilla-mobile/AutocompleteTextField#1